backend/data_tools/data/user_data.json5

```diff
@@ -137,8 +135,8 @@
         "POST_RESEARCH_PROJECT",
 
         "GET_USER",
-        "GET_USERS",
```

Comment:
am struggling to remember why we would have had GET_USERS (plural) in here

Reply:
See answer below